doc/gov/chapters/chapter05.md

```diff
@@ -1,18 +1,38 @@
 [<< Back](../)
 
-# 5. Chapter 5
+# 5. Concensus-based Decision Making
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
 * [5.1 Introduction](#5.1)
-* [5.2 Heading](#5.2)
-* [5.3 Heading](#5.3)
+* [5.2 Voting representatives](#5.2)
+* [5.3 Voting Process](#5.3)
 
 <a name="5.1"></a>
 ## 5.1 Introduction
+Consensus decision making is the model adopted by CNTT for all decisions taken within CNTT. This is essential for the success of CNTT and the buying needed from all parties involved in the community. In scenarios where disagreements are found, a reasonable effort shall be taken to identify the core cause of disagreements and work on a compromise that satisfies all parties involved in the discussion. 
+
+However, in scenarios where an agreement can not be reached, a voting process will be taken place to find a way forward. 
 
 <a name="5.2"></a>
-## 5.2 Heading
+## 5.2 Voting representatives
+
+Before any voting can take place, a voting representative list<sup>1</sup> shall be maintained that contains a list of individuals representing their corresponding organisation. Each organisation should only identify a single voting representative.
+
+Being on the voting representative list **doesn't** mean the right to vote, the right to vote will be identified via the following criteria<sup>2</sup>:
+
```

Comment:
Some changes are required. Example, "... must have people directly associated with it **who** physically attended .." instead of "... must have people directly associated with it physically attended ..."

More importantly, what is the value of  "created 2 Pull Requests"?

Reply:
@pgoyal01 text updated as per your input. please mark resolve if you are OK with the latest changes.

Reply:
@rabi-abdel Thanks